changelog/security/2022-02-09-vim.md

```diff
@@ -0,0 +1 @@
+- vim ([CVE-2021-3984](https://cve.circl.lu/cve/CVE-2021-3984), [CVE-2021-4019](https://cve.circl.lu/cve/CVE-2021-4019), [CVE-2021-4069](https://cve.circl.lu/cve/CVE-2021-4069), [CVE-2021-4136](https://cve.circl.lu/cve/CVE-2021-4136), [CVE-2021-4173](https://cve.circl.lu/cve/CVE-2021-4173),[ CVE-2021-4166](https://nvd.nist.gov/vuln/detail/CVE-2021-4166), [CVE-2021-4187](https://nvd.nist.gov/vuln/detail/CVE-2021-4187), [CVE-2021-4192](https://nvd.nist.gov/vuln/detail/CVE-2021-4192), [CVE-2021-4193](https://nvd.nist.gov/vuln/detail/CVE-2021-4193), [CVE-2022-0128](https://nvd.nist.gov/vuln/detail/CVE-2022-0128), [CVE-2022-0156](https://nvd.nist.gov/vuln/detail/CVE-2022-0156), [CVE-2022-0158](https://nvd.nist.gov/vuln/detail/CVE-2022-0158), [CVE-2022-0213](https://nvd.nist.gov/vuln/detail/CVE-2022-0213), [CVE-2022-0261](https://nvd.nist.gov/vuln/detail/CVE-2022-0261), [CVE-2022-0318](https://nvd.nist.gov/vuln/detail/CVE-2022-0318), [CVE-2022-0319](https://nvd.nist.gov/vuln/detail/CVE-2022-0319), [CVE-2022-0351](https://nvd.nist.gov/vuln/detail/CVE-2022-0351), [CVE-2022-0359](https://nvd.nist.gov/vuln/detail/CVE-2022-0359), [CVE-2022-0361](https://nvd.nist.gov/vuln/detail/CVE-2022-0361), [CVE-2022-0368](https://nvd.nist.gov/vuln/detail/CVE-2022-0368), [CVE-2022-0392](https://nvd.nist.gov/vuln/detail/CVE-2022-0392), [CVE-2022-0393](https://nvd.nist.gov/vuln/detail/CVE-2022-0393), [CVE-2022-0407](https://nvd.nist.gov/vuln/detail/CVE-2022-0407), [CVE-2022-0408](https://nvd.nist.gov/vuln/detail/CVE-2022-0408), [CVE-2022-0413](https://nvd.nist.gov/vuln/detail/CVE-2022-0413), [CVE-2022-0417](https://nvd.nist.gov/vuln/detail/CVE-2022-0417))
```

Comment:
Can you please replace `cve.circl.lu` with `nvd.nist.gov/vuln/detail`?

Can you please add [CVE-2022-0443](https://nvd.nist.gov/vuln/detail/CVE-2022-0443)?

Reply:
Good catch - did not notify `cve.circle.lu` and `nvd.nist.gov` were mixed :) Will add the latest one.

Reply:
@dongsupark thanks - it's done.